### ISSUE TITLE
vim-go 2.0 will drop support for Neovim?

### ISSUE THREAD

**Issue by preslavmihaylov** [fatih/vim-go#2530]
I was just looking at the vim-go milestones and I noticed the following note for the upcoming switch to version 2.0 of vim-go:
![image](https://user-images.githubusercontent.com/7642330/66023921-36461e80-e4fb-11e9-9ba7-35a5f4cddbc7.png)

I was wondering why that decision was taken, how it will impact NeoVim users, like myself, and if there are other vim-go users that are using nvim and might be affected?

**Comment by mnarrell** [comment]
There are absolutely other NeoVim users this would affect, if support is possibly dropped.

**Comment by bhcleek** [comment]
It's something we've talked about. There is no timeline for vim-go 2.0, and nothing is written in stone yet.

Neovim is not Vim. It's a Vim clone in the same way that Vim is a Vi clone, and it continues to diverge. The effort it takes to deal with the divergence is often quite frustrating, so dropping support for Neovim is something that Fatih and I discussed last year as a possibility for vim-go 2.0 if it ever happens.
